modules/seo-tools/jetpack-seo.php

```diff
@@ -6,6 +6,8 @@
 class Jetpack_SEO {
 	public function __construct() {
 		add_action( 'init', array( $this, 'init' ) );
+		add_action( 'init', array( Jetpack_SEO_Posts, 'register_post_meta' ), 20 );
```

Comment:
Getting PHP warning:
```
[22-Mar-2019 20:57:37 UTC] PHP Warning:  Use of undefined constant Jetpack_SEO_Posts - assumed 'Jetpack_SEO_Posts' (this will throw an Error in a future version of PHP) in /var/www/html/wp-content/plugins/jetpack/modules/seo-tools/jetpack-seo.php on line 9
[22-Mar-2019 20:57:37 UTC] PHP Stack trace:
[22-Mar-2019 20:57:37 UTC] PHP   1. {main}() /var/www/html/wp-admin/admin-ajax.php:0
[22-Mar-2019 20:57:37 UTC] PHP   2. require_once() /var/www/html/wp-admin/admin-ajax.php:22
[22-Mar-2019 20:57:37 UTC] PHP   3. require_once() /var/www/html/wp-load.php:37
[22-Mar-2019 20:57:37 UTC] PHP   4. require_once() /var/www/html/wp-config.php:84
[22-Mar-2019 20:57:37 UTC] PHP   5. do_action() /var/www/html/wp-settings.php:374
[22-Mar-2019 20:57:37 UTC] PHP   6. WP_Hook->do_action() /var/www/html/wp-includes/plugin.php:465
[22-Mar-2019 20:57:37 UTC] PHP   7. WP_Hook->apply_filters() /var/www/html/wp-includes/class-wp-hook.php:310
[22-Mar-2019 20:57:37 UTC] PHP   8. Jetpack::load_modules() /var/www/html/wp-includes/class-wp-hook.php:286
[22-Mar-2019 20:57:37 UTC] PHP   9. include_once() /var/www/html/wp-content/plugins/jetpack/class.jetpack.php:1858
[22-Mar-2019 20:57:37 UTC] PHP  10. Jetpack_SEO->__construct() /var/www/html/wp-content/plugins/jetpack/modules/seo-tools.php:3
```